src/test/java/core/basesyntax/services/impl/ReportWriterImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.jupiter.api.Assertions.assertLinesMatch;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.services.ReportWriter;
+import java.io.File;
+import java.util.StringJoiner;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportWriterImplTest {
+    private static final String TITLE = "fruit,quantity";
+    private static final String APPLE = "apple";
+    private static final String BANANA = "banana";
+    private static final String APPLE_WITH_QUANTITY = "apple,20";
+    private static final String BANANA_WITH_QUANTITY = "banana,50";
+    private static final File FINAL_REPORT
+            = new File("src/main/resources/reports/final-report.csv");
```

Comment:
a bit weird you have string filenams for others as constants, and this one is a File